src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -126,10 +140,31 @@
                 } else {
                     return "Select ADR dataset"
                 }
+            },
+            outOfDateResources() {
+                if (!this.selectedDataset) {
+                    return {};
+                }
+                const resources = this.selectedDataset.resources;
+                const outOfDateResources: { [k in keyof DatasetResourceSet]?: true } = {};
+                Object.keys(resources).map((k) => {
+                    const key = k as keyof DatasetResourceSet;
+                    if (resources[key] && resources[key]!!.outOfDate) {
+                        outOfDateResources[key] = true;
+                    }
+                });
+                return outOfDateResources
```

Comment:
this data structure is going to make it easy to refresh relevant files,  e.g. 
`this.outOfDateResources["pjnz"] && importPjnz()`